docker/docker-compose.yml

```diff
@@ -42,6 +42,14 @@ services:
     environment:
       SW_STORAGE: elasticsearch
       SW_STORAGE_ES_CLUSTER_NODES: elasticsearch:9200
+      SW_HEALTH_CHECKER: default
+      SW_TELEMETRY: prometheus
+    healthcheck:
+      test: ["CMD", "./bin/swctl", "ch"]
```

Comment:
I didn't found this command doc on the CLI repo. Seems we missed it. Please update the doc. @hanahmily